src/common/test/MockApolloClientProvider.tsx

```diff
@@ -23,8 +25,13 @@ export function MockApolloClientProvider({
   responseProvider: ResponseProvider;
 }): React.ReactElement {
   const customGlobal = global as GlobalWithFetchMock;
```

Comment:
I see that the jest-fetch-mock library is updated in this pr from 2.1.2 to 3.0.3 version. In the documentation of 3.0.3 jest-fetch-mock (https://www.npmjs.com/package/jest-fetch-mock) no longer mention this type _ GlobalWithFetchMock_. So I wonder if the preferred way of handling globals with the jest-fetch-mock library has changed?

Reply:
GlobalWithFetchMock still exists, but you're right, documentation has changed regarding the setup. I'll try to remove  GlobalWithFetchMock and use enableMocks()

Reply:
Global stuff removed!